packages/loader/driver-utils/src/toBuffer.ts

```diff
@@ -0,0 +1,14 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IsoBuffer } from "@fluidframework/common-utils";
+
+/**
+ * Convert base64 or utf8 string to array buffer
+ */
+export function toBuffer(input: string, encoding: string): ArrayBufferLike {
```

Comment:
Can you please put @deprecated comment and reference to everyone using implementation from common-utils?
Opening an issue to actually clean it in release would be great!